NetworkOfPrivateClinics/NetworkOfPrivateClinics/Doctor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NetworkOfPrivateClinics.CustomExceptions;
+using NetworkOfPrivateClinics.Exceptions;
+
+namespace NetworkOfPrivateClinics
+{
+    public enum DoctorType
+    {
+        Neurologist,
+        Gastroenterologist,
+        Cardiology,
+        Oncologist,
+        Psychiatrist,
+        Dermatologist,
+        Anasthesiologist
+    }
+
+    public class Doctor
+    {
+        public static int DoctorID { get; private set; }
+        public string Name { get; private set; }
+        public string Surname { get; private set; }
+        public DoctorType Type { get; private set; }
+        public double CostOfAdmission { get; private set; }
```

Comment:
We use `decimal` for strict and accurate values. double and float for fast calculation, f.e. vectors